charts/csi-cloudscale/templates/crds.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.installCRDs }}
```

Comment:
Are you still using a version of `csi-cloudscale < 2.0.0`?
Do you think we should support the old versions in the helm Chart? If yes, we should probably, also support upgrading.